### PR TITLE
chore: remove some commented code and simplify some

### PR DESCRIPTION
**Pull request by hezhizhen**
### Ⅰ. Describe what this PR does

1. remove commented code
2. tweak imports order (automatically done by GoLand)
3. use existing functions to replace duplicated parts

### CODE REVIEW

**Review by FillZpp — COMMENTED** [review]
LGTM, thanks for contributing! One nit:

File: pkg/webhook/util/configuration/configuration.go [hunk 1]
```diff
@@ -73,10 +73,12 @@ func Ensure(kubeClient clientset.Interface, handlers map[string]admission.Handle
 		if wh.ClientConfig.Service != nil {
 			wh.ClientConfig.Service.Namespace = webhookutil.GetNamespace()
 			wh.ClientConfig.Service.Name = webhookutil.GetServiceName()
+
+			if host := webhookutil.GetHost(); len(host) > 0 {
+				convertClientConfig(&wh.ClientConfig, host, webhookutil.GetPort())
+			}
 		}
-		if host := webhookutil.GetHost(); len(host) > 0 && wh.ClientConfig.Service != nil {
```
[the remark this answers is not in the document]
Comment (hezhizhen) [reply pkg/webhook/util/configuration/configuration.go:77]: Sure.

[the remark this answers is not in the document]
Comment (hezhizhen) [reply pkg/webhook/util/configuration/configuration.go:77]: Fixed it and PTAL.

**Comment by FillZpp** [comment]
It seems UT failed because of `go.mod` changed after make command. Please `make generate manifests` and then push again.

**Comment by FillZpp** [comment]
@hezhizhen Hey, could you fix this today, for we are preparing for v1.2 release and I hope this can be ready for it.

**Comment by hezhizhen** [comment]
> @hezhizhen Hey, could you fix this today, for we are preparing for v1.2 release and I hope this can be ready for it.

I tried yesterday, but nothing's different. I guess that it is because my go version is 1.18. I'll fix it today.

**Comment by FillZpp** [comment]
> I tried yesterday, but nothing's different. I guess that it is because my go version is 1.18. I'll fix it today.

Oh it might be. We have seen go versions always make these problems. Can `go mod tidy` do any help?

**Comment by hezhizhen** [comment]
> > I tried yesterday, but nothing's different. I guess that it is because my go version is 1.18. I'll fix it today.
> 
> Oh it might be. We have seen go versions always make these problems. Can `go mod tidy` do any help?

Screen Shot 2022-05-27 at 11 49 50
 
It seems that go 1.17 and 1.18 handle dependencies differently. 1.17 treats github.com/go-openapi/spec as indirect and 1.18 treats it as direct.

Is there a way to fix go version for this project to avoid cases like this?

**Comment by FillZpp** [comment]
> Is there a way to fix go version for this project to avoid cases like this?

Emm... for this case, putting `_ "github.com/go-openapi/spec"` into `hack/tools.go` explicitly might solve the problem. Not sure if there can be some better ways.

**Comment by hezhizhen** [comment]
> > Is there a way to fix go version for this project to avoid cases like this?
> 
> Emm... for this case, putting `_ "github.com/go-openapi/spec"` into `hack/tools.go` explicitly might solve the problem. Not sure if there can be some better ways.

For this case, yes; for general problems caused by different go versions, no.

**Review by FillZpp — APPROVED** [review]
/lgtm

**Pull request merged**